includes/wikia/ExternalUser_Wikia.php

```diff
@@ -1,6 +1,12 @@
 <?php
 
+use Wikia\Logger\WikiaLogger;
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\User\Auth\AuthServiceAccessor;
+
 class ExternalUser_Wikia extends ExternalUser {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "ExternalUser_Wikia" has 30 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename class "ExternalUser_Wikia" to match the regular expression ^[A-Z][a-zA-Z0-9]*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS101)